classes/autoptimizeConfig.php

```diff
@@ -539,4 +551,46 @@ static function ao_admin_tabs(){
         
         return $tabContent;
     }
+
+	public function load_custom_wp_admin_style()
+	{
+		wp_enqueue_style( 'autoptimize-toolbar', plugins_url( 'autoptimize/css/toolbar.css' ), array(), time(), "all" );
+	}
+    
+	public function load_custom_wp_admin_javascript()
+	{
+		wp_enqueue_script( 'autoptimize-toolbar', plugins_url( 'autoptimize/js/toolbar.js' ), array(), time(), true );
+	}
+	
+	public function load_admin_toolbar()
+	{
+		$show = false;
+
+		// Admin
+		$show = ( current_user_can( 'manage_options' ) || current_user_can( 'edit_others_pages' ) ) ? true : false;
+
+		// Author
+		if( defined( 'AUTOPTIMIZE_TOOLBAR_FOR_AUTHOR' ) && AUTOPTIMIZE_TOOLBAR_FOR_AUTHOR )
```

Comment:
still; what kind of changes done by an editor would warrant the autoptimize cache having to be purged? why not just
`
$show = ( current_user_can( 'manage_options' ));
`
?